addons/website/data/website_contactus_data.xml

```diff
@@ -3,142 +3,126 @@
 
     <template id="website.contactus" name="Contact Us">
         <t t-call="website.layout">
-            <div id="wrap">
-                <div class="oe_structure">
-                    <section class="s_title parallax s_parallax_is_fixed bg-black-50 pt24 pb24" data-vcss="001" data-snippet="s_title" data-scroll-background-ratio="1">
-                        <span class="s_parallax_bg oe_img_bg" style="background-image: url('/web/image/website.s_banner_default_image'); background-position: 50% 0;"/>
-                        <div class="o_we_bg_filter bg-black-50"/>
-                        <div class="container">
-                            <h1>Contact us</h1>
-                        </div>
-                    </section>
-                </div>
-                <div class="container mt-4 mb-5">
-                    <div class="row">
-                        <div class="col-lg-7 col-xl-6 mr-lg-auto">
-                            <div class="oe_structure">
-                                <section class="s_text_block" data-snippet="s_text_block">
+            <span class="hidden" data-for="contactus_form" t-att-data-values="{'email_to': res_company.email}"/>
+            <div id="wrap" class="oe_structure">
```

Comment:
Is not there an `oe_empty` class alongside the `oe_structure` class in default pages?